CONTRIBUTING.md

```diff
@@ -15,10 +15,10 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 ### Additional Helpful Links for Contributors  
 Many people all over the world have helped make this project better.  You'll want to check out:
 
-* [What are some good first issues for new contributors to the repo?](https://github.com/azure/azure-sdk-for-c/issues?q=is%3Aopen+is%3Aissue+label%3A%22up+for+grabs%22)
-* [How to build and test your change][azure_sdk_for_c_contributing_developer_guide]
-* [How you can make a change happen!][azure_sdk_for_c_contributing_pull_requests]
-* Frequently Asked Questions (FAQ) and Conceptual Topics in the detailed [Azure SDK for Embedded C wiki](https://github.com/azure/azure-sdk-for-c/wiki).
+* [What are some good first issues for new contributors to the repo?](https://github.com/azure/azure-sdk-for-c/issues?q=is%3Aopen+is%3Aissue+label%3Aup-for-grabs)
+* [How to build and test your change](https://github.com/Azure/azure-sdk-for-c/blob/master/CONTRIBUTING.md#developer-guide)
+* [How you can make a change happen!](https://github.com/Azure/azure-sdk-for-c/blob/master/CONTRIBUTING.md#pull-requests)
+* Conceptual Topics in the detailed [Azure SDK for Embedded C docs](https://azure.github.io/azure-sdk-for-c).
```

Comment:
The wiki is outdated and maybe we should remove it or just link to the published docs from there.

Reply:
Thanks Ahson for checking all this! Yes, if links are dead, let's have this fixed either way on the completion of this PR.